packages/block-library/src/template-part/edit/use-template-part-post.js

```diff
@@ -26,7 +26,7 @@ export default function useTemplatePartPost( postId, slug, theme ) {
 					'postType',
 					'wp_template_part',
 					{
-						status: 'auto-draft',
+						status: [ 'publish', 'auto-draft' ],
```

Comment:
Needs a rebase I think? #21766 [has this](https://github.com/WordPress/gutenberg/pull/21766/files#diff-13317c691c7576258126440fe8df98b8R29).

Reply:
Nice, that one merged.  I rebased this =)